### ISSUE TITLE
Membership: clicking on discard button doesn't do anything

### ISSUE THREAD

**Issue by fmichonneau** [carpentries/amy#1838]
when editing persons or members in consortium view:
1. click on "Add another"
2. click on "Discard" 

what happens:
- the background becomes grey

what I would expect:
- the 2 entries for the form to disappear from the screen

Note that clicking "Save" after clicking "Discard" works as expected (even if the 2 form entries are filled, the person/member is not added to the membership).

**Comment by maneesha** [comment]
Just a note that this applies to the test server for the membership infrastructure, not the live AMY server.

**Comment by pbanaszkiewicz** [comment]
@fmichonneau @maneesha  This is not a bug, but it can be unexpected behavior. Let me explain how this works:

The "Discard" button appears only after clicking "Add another" and applies to the single "small" form that is being duplicated. For example, see below: I clicked 3 times "Add another" and three smaller forms appeared.
![obraz](https://user-images.githubusercontent.com/72821/111078777-73d41700-84f7-11eb-87d1-69479f10ab1b.png)

Now, if I decide I don't want to work with specific form, I will click "Discard" so that form becomes unavailable. I wanted to disable all fields in that form one by one but it turned out difficult since we rely on [3rd party widget](select2.org). Instead I grey-out the background to indicate the form won't be used. See below for example with first and last form discarded:
![obraz](https://user-images.githubusercontent.com/72821/111078892-f78e0380-84f7-11eb-8d26-04867e3b9cae.png)

Not only this greys out the forms, but it also disables the form validation and makes it not taking part in process.


To summarise I understand that Discard button may not behave like you expected. Please read my explanation above and provide behaviour you would like to see when you discard single form from multiform page.

**Comment by fmichonneau** [comment]
Thanks Piotr, your explanation clears things up for me, and I'm happy to leave this as is.

**Comment by maneesha** [comment]
Re-opening per my conversation with @pbanaszkiewicz  today.
When a user hits "Discard" the form changes to gray, but the form itself is not disabled and whatever was filled in remains (not as expected). If the user then hits "save" the discarded option does not save (as expected). 

This should change so that when the user hits "Discard" the discarded form item disappears.
